app/models/model.js

```diff
@@ -5,27 +5,27 @@ const queryBuilder = require('knex')(process.env.NODE_ENV === 'development' ? de
 
 class Model {
```

Comment:
Remove this and use raw queries by knex. This will make the codebase more clear.